src/pages/index.js

```diff
@@ -13,26 +15,25 @@ import {
   profileEditModal
 } from "../utils/constants.js";
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a5172d76-7206-4bb6-b609-2e8e17b83724",
+    "Content-Type": "application/json",
+  },
+});
+
 function handleImageClick(cardData) {
   popupWithImage.open(cardData);
 }
 
-function createCard(item) {
-  const cardElement = new Card(item, "#card-template", handleImageClick);
-  return cardElement.getView()
+function renderCard(item) {
+  const card = new Card(item, "#card-template", handleImageClick, handleDeleteButton, handleLikeClick);
+  return card.getView()
 }
 
-function renderCard(cardData) {
-  const card = createCard(cardData);
-  section.addItem(card);
-}
 
-function handleAddCardFormSubmit(inputValues) {
-  renderCard(inputValues);
-  variable.addCardForm.reset();
-  cardFormFormValidator.changeButtonState();
-  cardPopup.close();
-}
+
 
 //FormValidator
 const profileEditModalFormValidator = new FormValidator(profileEditModal,config);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```